app/extensions/brave/locales/en-US/extensions.properties

```diff
@@ -29,3 +29,4 @@ torrent= Torrent Viewer
 torrentDesc=Uses WebTorrent to display torrents directly in the browser. Supports torrent files and magnet links.
 vimium= Vimium
 vimiumDesc=
+honeyDesc=Automatically find and apply coupon codes when you shop online!
```

Comment:
you need a name here as well, like `honey=Honey`. If the extension has a different name like pocket (which shows as "Save to Pocket"), you'll need to use the same first letter for the variable as well. Otherwise, we can't sort by name in extensions table.